src/frontend/src/lib/components/dapps/DappPromoBanner.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+	import Button from '$lib/components/ui/Button.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { DApp } from '$lib/types/dapp';
+	export let dApp: DApp;
+</script>
+
+<div
+	class="flex h-64 items-end overflow-hidden rounded-2xl bg-cover bg-center"
+	style="background-image: url('{dApp.imageUrl}')"
+>
+	<div class="flex-1 px-4 py-4 backdrop-blur-sm">
+		<div class="flex items-center gap-x-2">
+			<img class="h-12 w-12 rounded-full" src={dApp.logoUrl} alt={dApp.name} />
+			<div class="flex-1">
+				<h6 class="text-sm font-bold text-info">{$i18n.dapps.text.featured}</h6>
+				<h4 class="text-white">{dApp.name}</h4>
+			</div>
+			<a href={dApp.url} target="_blank" class="no-underline">
+				<Button size="sm" colorStyle="secondary" styleClass="flex-grow-0"
```

Comment:
As far as I know, a link should not correct a button. I think this is not conform with the web specification.